language/stdlib/modules/libra_system.move

```diff
@@ -72,6 +72,16 @@ module LibraSystem {
         });
     }
 
+    public fun grant_reconfiguration_capability(): ReconfigurationCapability {
```

Comment:
Do we want to ensure that this is a singleton?

Reply:
My intention is actually not make it as singleton, as I was imagining that with the deployment of newer modules there can be other functionalities that requires a reconfiguration.

Reply:
Allowing an unbounded number of these to be created + stored anywhere will make it really difficult to reason about the conditions under which reconfiguration can occur. I would strongly prefer if there was a single instance of this capability. This will allow us to determine when reconfiguration can occur by simply looking at who can access the capability and when.